snippets/messaging.go

```diff
@@ -136,8 +136,72 @@ func sendAll(ctx context.Context, client *messaging.Client) {
 
 	// See the BatchResponse reference documentation
 	// for the contents of response.
-	fmt.Println(br.SuccessCount, " messages were sent successfully")
-	// [END send_a_batch_golang]
+	fmt.Printf("%d messages were sent successfully\n", br.SuccessCount)
+	// [END send_all]
+}
+
+func sendMulticast(ctx context.Context, client *messaging.Client) {
+	// [START send_multicast]
+	// Create a list containing up to 100 registration tokens.
+	// This registration tokens come from the client FCM SDKs.
+	registrationTokens := []string{
+		"YOUR_REGISTRATION_TOKEN_1",
+		// ...
+		"YOUR_REGISTRATION_TOKEN_n",
+	}
+	message := &messaging.MulticastMessage{
+		Data: map[string]string{
+			"score": "850",
+			"time":  "2:45",
+		},
+		Tokens: registrationTokens,
+	}
+
+	br, err := client.SendMulticast(context.Background(), message)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// See the BatchResponse reference documentation
+	// for the contents of response.
+	fmt.Printf("%d messages were sent successfully\n", br.SuccessCount)
+	// [END send_multicast]
+}
+
+func sendMulticastAndHandleErrors(ctx context.Context, client *messaging.Client) {
+	// [START send_multicast_error]
+	// Create a list containing up to 100 registration tokens.
+	// This registration tokens come from the client FCM SDKs.
+	registrationTokens := []string{
+		"YOUR_REGISTRATION_TOKEN_1",
+		// ...
+		"YOUR_REGISTRATION_TOKEN_n",
+	}
+	message := &messaging.MulticastMessage{
+		Data: map[string]string{
+			"score": "850",
+			"time":  "2:45",
+		},
+		Tokens: registrationTokens,
+	}
+
+	br, err := client.SendMulticast(context.Background(), message)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	if br.FailureCount > 0 {
+		var failedTokens []string
+		for idx, resp := range br.Responses {
+			if !resp.Success {
+				// The order of responses corresponds to the order of the registration tokens
```

Comment:
1. You might want to double check that this line isn't too long to look right on the website. A lot of the other comments are broken up to make short lines.

2. All of the other sentences in comments in this doc end with a period, so I'd recommend adding one here for consistency.

Reply:
1. Line length looks ok (87 columns). It's about the same length in case of the Java example.
2. Done.